app/cdap/components/Connections/AddConnectionBtnModal/index.tsx

```diff
@@ -35,9 +46,9 @@ export default function AddConnectionBtnModal({ onCreate, className = null }) {
   return (
     <>
       <Button
-        variant="contained"
+        variant="outlined"
         color="primary"
-        className={className}
+        className={clsx(className, classes.root)}
```

Comment:
Is setting the style necessary? An outlined primary button should have a white background already: https://material-ui.com/components/buttons/#outlined-buttons

In general, we don't want to custom-code buttons. We want to get to a small number of pre-defined buttons.

Reply:
@njbriggs the background color is set in the other add connection button and without it looks like this (no background color)
<img width="262" alt="Screen Shot 2021-07-20 at 3 05 37 PM" src="https://user-images.githubusercontent.com/2054644/126401437-1b2dab5c-6f8f-4ed3-be50-7f2175c70470.png">

Reply:
Ack, you're right.